test/proposal.test.ts

```diff
@@ -111,11 +111,11 @@ describe('Pod.getProposals', () => {
     );
   });
 
-  test('Pod.getProposals({ status: executed }) will return only the active proposal', async () => {
+  test('Pod.getProposals({ status: passed || rejected }) will return only the active proposal', async () => {
```

Comment:
I don't think this should be the functionality. This is also a breaking change. I think we can leave the status as 'executed' and it still returns all passed and rejected proposals. Thoughts? @jtlacci

Reply:
i agree, also would prefer we didn't introduce breaking changes